packages/bouncer-deps/build

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# This is for installing bouncer dependencies that are required
+# in both, upstream (variant 'default') and downstream (variant
+# 'ee') bouncer.
+
+set -exu
+
+source /opt/mesosphere/environment.export
+export LIB_INSTALL_DIR="$PKG_PATH/lib/python3.6/site-packages"
+mkdir -p "$LIB_INSTALL_DIR"
+
+echo "pip version: $(pip --version)"
+echo "Install packages from wheel files."
+pip install --no-deps --no-index --target="$LIB_INSTALL_DIR" /pkg/src/jsonschema/*.whl
+
+# Resolve 'Package libffi was not found in the pkg-config search path'.
+export PKG_CONFIG_PATH=/opt/mesosphere/lib/pkgconfig
+
+# Install from local source directories (git checkout or extracted tarballs).
+# Order matters. Some of these packages require Cython or build C extensions
+# for speed-up if Cython is availble.
```

Comment:
> And where is Cython installed?

At some point during the last years I have made it so that Cython is installed as part of the `python` pkgpanda package; so that it's "always available" for other pkgpanda packages that define `python` as a dependency.

Reply:
> Is the "Order matters" comment related to the Cython comment? 

I don't think so. I think I have made this comment when I first set up the Bouncer pkgpanda package back in 2015.

I think it comes from the fact that we install package after package in "offline mode", and some package installations might fail if a certain dependency is not there yet.

Reply:
(while I would love to know better right now, I would trust past JP)